lib/config.js

```diff
@@ -631,6 +631,7 @@ const tryToLoad = (configPath, fallbackHandler) => {
   try {
     pkg = require(sysPath.resolve('.', 'package.json'));
   } catch (e) {
+    logger.error(e);
```

Comment:
Just adding a comment or something in there should have made eslint happy

like

```
} catch (e) {
  // no-op
}
```